### ISSUE TITLE
Contacts, Campaigns, Statistics: Not working, only generic "Error" message

### ISSUE THREAD

**Issue by tiktaalik-dev** [mailjet/joomla-mailjet-plugin-apiv3#35]
Hello there,
Just installed the plugin in an up-to-date (3.9.24) version of Joomla.
"Settings" page of the component works fine, it also changed my SMTP settings correctly and sent a test email successfully. Module also works well. But the Contacts, Campaign, and Statistics page simply do not load anything. I only get a generic message stating the word "Error"!
I checked my website and server error logs in case there might be a PHP error trace but they are completely empty.
I also checked on my browser DevTools inspector but found nothing there either. No console error messages, no hidden elements, nothing at all. Just the word Error as the sole content.
Can you check on it, please?
Thanks in advance for your help! :-)

**Comment by SemaphoreOxalis** [comment]
I think this module don't work anymore since mailjet have been buyout again and again...

**Comment by tiktaalik-dev** [comment]
@SemaphoreOxalis Yeah, I reckon that too. Almost a year later and they never replied, that's sad.
I didn't understand the code last time I checked it up. I reckon, I'll just give it another go one of these days.
Cheers
